### ISSUE TITLE
Support VAST 4.0

### ISSUE THREAD

**Issue by ghost** [MailOnline/videojs-vast-vpaid#254]
```
validateVASTTree, not supported VAST version "4"
```

Please add support for VAST version 4.

**Comment by Fetz** [comment]
@JHGitty, yes we are aware of the issue, since VAST 4.0, is a bit different and a bit more complex we are thinking of breaking this modules in different parts to be easier to upgrade

**Comment by AndreasHansenATG** [comment]
Hello Fetz, do you have new information on progress regarding VAST4 support? We are seeing growing demand for VAST4 compatibility and looking for a way to realize it with videoJS.

**Comment by kyungw00k** [comment]
@AndreasHansenATG 

Recently, I add some functionality to parser based on the parser part of this project.

https://github.com/kyungw00k/vast-parser

I haven't tested 4.0 response but I did some 3.0 responses.
